nvflare/app_opt/xgboost/data_loader.py

```diff
@@ -21,7 +21,7 @@
 
 class XGBDataLoader(ABC):
     @abstractmethod
-    def load_data(self, client_id) -> Tuple[xgb.core.DMatrix, xgb.core.DMatrix]:
+    def load_data(self) -> Tuple[xgb.core.DMatrix, xgb.core.DMatrix]:
         """Loads data for xgboost.
 
         Returns:
```

Comment:
If change the method signature here, need to also change all other places that use this class.
For example horizontal xgboost both histogram and tree examples.

I think we should keep this signature as it is

Reply:
``load_data(client_id: str, app_dir: str):``